src/Service/Mailer/NotificationMail.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace App\Service\Mailer;
+
+use App\Entity\Signalement;
+use App\Entity\Territory;
+use App\Entity\User;
+
+class NotificationMail
+{
+    public function __construct(
+        private readonly NotificationMailerType $type,
+        private readonly array|string $to,
+        private readonly ?string $fromEmail = null,
+        private readonly ?string $fromFullname = null,
+        private readonly ?string $message = null,
+        private readonly ?Territory $territory = null,
+        private readonly ?User $user = null,
+        private readonly ?Signalement $signalement = null,
+        private readonly ?\Throwable $event = null,
+        private readonly mixed $attachment = null,
+        private readonly ?string $motif = null,
+        private readonly ?string $cronLabel = null,
+        private readonly ?int $cronCount = null,
+        private readonly array $params = [],
+    ) {
```

Comment:
Erreur à la validation d'un signalement : 

`App\Service\Mailer\NotificationMail::__construct(): Argument #9 ($event) must be of type ?Throwable, Symfony\Component\HttpKernel\Event\ExceptionEvent given, called in /app/src/EventListener/ExceptionListener.php on line 40`

Reply:
Même erreur au refus d'un signalement

Reply:
(je ne sais pas pourquoi mon commentaire s'est mis sur ce fichier par contre)

Reply:
C'est corrigé mais j'ai pas compris ton dernier commentaire

Reply:
t'inquiète :)

Reply:
c'est bon